cmd/util/cmd/verify_execution_result/cmd.go

```diff
@@ -0,0 +1,100 @@
+package verify
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/engine/verification/verifier"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var (
+	flagLastK            uint64
+	flagDatadir          string
+	flagChunkDataPackDir string
+	flagChain            string
+	flagFromTo           string
+)
+
+// # verify the last 100 sealed blocks
+// ./util verify_execution_result --chain flow-testnet --datadir /var/flow/data/protocol --chunk_data_pack_dir /var/flow/data/chunk_data_pack --lastk 100
+// # verify the blocks from height 2000 to 3000
+// ./util verify_execution_result --chain flow-testnet --datadir /var/flow/data/protocol --chunk_data_pack_dir /var/flow/data/chunk_data_pack --from_to 2000-3000
+var Cmd = &cobra.Command{
+	Use:   "verify-execution-result",
+	Short: "verify block execution by verifying all chunks in the result",
+	Run:   run,
+}
+
+func init() {
+	Cmd.Flags().StringVar(&flagChain, "chain", "", "Chain name")
+	_ = Cmd.MarkFlagRequired("chain")
+
+	Cmd.Flags().StringVar(&flagDatadir, "datadir", "/var/flow/data/protocol",
+		"directory that stores the protocol state")
+	_ = Cmd.MarkFlagRequired("datadir")
+
+	Cmd.Flags().StringVar(&flagChunkDataPackDir, "chunk_data_pack_dir", "/var/flow/data/chunk_data_pack",
+		"directory that stores the protocol state")
+	_ = Cmd.MarkFlagRequired("chunk_data_pack_dir")
+
+	Cmd.Flags().Uint64Var(&flagLastK, "lastk", 1,
+		"last k sealed blocks to verify")
+
+	Cmd.Flags().StringVar(&flagFromTo, "from_to", "",
+		"the height range to verify blocks (inclusive), i.e, 1-1000, 1000-2000, 2000-3000, etc.")
+}
+
+func run(*cobra.Command, []string) {
+	chainID := flow.ChainID(flagChain)
```

Comment:
We need `chainID.Chain()` to validate received `flagChain`.

```suggestion
	chainID := flow.ChainID(flagChain)
	_ = chainID.Chain()
```